lib/apiservers/engine/backends/backends.go

```diff
@@ -98,6 +99,11 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 		return err
 	}
 
+	// creates and potentially restore repository cache
+	if err := cache.NewRepositoryCache(portLayerClient); err != nil {
```

Comment:
Is it viable to run this after we launch the goroutines? I think `Init` blocking until this is done is correct, but no reason not to do image and container work in parallel that I know of.